lib/Targets/ADFGenerateCppGraph.cpp

```diff
@@ -92,8 +93,8 @@ struct GraphWriter {
       return "stream";
     else if (auto windowType = type.dyn_cast<ParameterType>())
       return "parameter";
-    else
-      assert(false);
+    assert(false);
```

Comment:
Just curious on these trivial use cases.
Is it not a godd use case for `std::terminate()`? What else if the code goes on here?